database-mysql/db-helpers.js

```diff
@@ -0,0 +1,10 @@
+cost Users = require('./index');
```

Comment:
did you test this helper? line 1 and 2 error because you forgot the n in const
please always test your code before pushing.
you can simply run this file to test
add a temporary call to the helper to see if it returns the proper results from db